test/prototype_common_utils.py

```diff
@@ -313,6 +315,38 @@ def make_image_loaders(
 make_images = from_loaders(make_image_loaders)
 
 
+def make_image_loader_for_interpolation(size="random", *, color_space=features.ColorSpace.RGB, dtype=torch.uint8):
+    size = _parse_spatial_size(size)
+    num_channels = get_num_channels(color_space)
+
+    def fn(shape, dtype, device):
+        num_channels, height, width = shape
+
+        top = int((0.8 * height) // 2)
+        left = int((0.8 * width) // 2)
+        bottom = height - top
+        right = width - left
+
+        image_pil = PIL.Image.new("L", (width, height))
+        draw = PIL.ImageDraw.Draw(image_pil)
+        draw.ellipse(((left, top), (right, bottom)), fill=255)
+
+        return features.Image(to_image_tensor(image_pil).repeat(num_channels, 1, 1), color_space=color_space).to(
+            dtype=dtype, device=device
+        )
+
+    return ImageLoader(fn, shape=(num_channels, *size), dtype=dtype, color_space=color_space)
+
+
+def make_image_loaders_for_interpolation(
+    sizes=((233, 147),),
```

Comment:
I didn't use the default shapes here since they are pretty small and thus the area of the border of the ellipse is quite large compared to the overall image and thus larger tolerances would need to be larger. Open for discussion about this.